### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by the-icarus**
Updated the gatsby auto generated templated with a more project related readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,97 +1,27 @@
...
-- **To dive straight into code samples, head [to our documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
-
-## 💫 Deploy
-
-[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/gatsbyjs/gatsby-starter-hello-world)
-
-<!-- AUTO-GENERATED-CONTENT:END -->
+- æternity:
+  - [ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV](https://explorer.aepps.com/#/account/ak_5z1fmzTKR1GA1P7qiLDCC1s3V7AK2RRpNbXqUhfHQbUeg7mmV)
+  - [kryptokrauts.test](https://www.aeknow.org/aens/query/kryptokrauts.test)
```
Comment (marc0olo) [inline README.md]: please remove the AENS domain. in the upcoming hardfork the `.test` namespaces will be purged. we can add it later on again

File: README.md [hunk 2]
```diff
@@ -1,97 +1,27 @@
...
 <h1 align="center">
-  Gatsby's hello-world starter
+  kryptokrauts.com
 </h1>
 
-Kick off your project with this hello-world boilerplate. This starter ships with the main Gatsby configuration files you might need to get up and running blazing fast with the blazing fast app generator for React.
+This is the repository for the kryptokrauts.com homepage.  
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.org/docs/gatsby-starters/)._
+We are a group of Blockchain enthusiasts building a community for Blockchain and Crypto.
```
Comment (marc0olo) [inline README.md]: please all words in lower letters =)

**Review by marc0olo — CHANGES_REQUESTED** [review]
@the-icarus can you go through the 2 comments and make the small changes? thanks

File: README.md (hunk 1 above)
Comment (the-icarus) [reply README.md]: Updated. I also removed the domain from the original article (2019-07-10-aepp-sdk-v1.2.0-released.md)

File: README.md (hunk 2 above)
Comment (the-icarus) [reply README.md]: Done :)

**Review by marc0olo — APPROVED** [review]

**Pull request merged**